### PR TITLE
Auth fix

### PR DESCRIPTION
**Pull request by Atik-hridoy**

### CODE REVIEW

File: lib/src/screens/registration_screen.dart [hunk 1]
```diff
@@ -27,7 +27,6 @@ class _RegistrationScreenState extends State<RegistrationScreen> {
         _emailController.text.isEmpty ||
         _passwordController.text.isEmpty ||
         _confirmPasswordController.text.isEmpty) {
-      //_showToast("Please fill in all fields.");
```
Comment (FahimHoque) [inline lib/src/screens/registration_screen.dart]: always delete unused code. Master will remain comment free

**Review by Atik-hridoy — COMMENTED** [review]
Screenshot from 2023-11-30 19-53-57
Screenshot from 2023-11-30 19-55-29

I fixed the Authentication by change the realtime firestores rule.

**Comment by FahimHoque** [comment]
changes looks good. feel free to merge. Remember to `git pull` from master

**Review by FahimHoque — APPROVED** [review]
1. remove codes/folders related to windows, web, macos and linux
2. better commit messages are expected

**Pull request merged**